PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -61,3 +68,12 @@
                                                            electronSource = "gedGsfElectrons",
                                                            genParticleMatch = "electronMatch"
                                                            )
+
+# Schedule rekeying of seed BDT ValueMaps by reco::GsfElectron for run2_miniAOD_UL
+from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeedValueMaps_cff import rekeyLowPtGsfElectronSeedValueMaps
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import lowPtGsfElectronID
+makePatLowPtElectronsTask_ = makePatLowPtElectronsTask.copy()
```

Comment:
```suggestion
_makePatLowPtElectronsTask = makePatLowPtElectronsTask.copy()
```
so that this name is not exported by default

Reply:
Done.

Reply:
Done.